internal/pkg/resource_availability_manager/main.go

```diff
@@ -19,10 +20,15 @@ func NewResourceAvailabilityManager(settings ResourceAvailabilitySettings) *Reso
 }
 
 type NetworkPointType string
+type ResourceAvailabilityRequestType string
 
 const (
-	NetworkPointIngress NetworkPointType = "ingress"
-	NetworkPointService NetworkPointType = "service"
+	NetworkPointIngress                     NetworkPointType                = "ingress"
+	NetworkPointService                     NetworkPointType                = "service"
+	ResourceAvailabilityRequestIngress      ResourceAvailabilityRequestType = "ingress"
+	ResourceAvailabilityRequestLoadbalancer ResourceAvailabilityRequestType = "loadbalancer"
+	FailedHttpCode                          int                             = 500
```

Comment:
Any response code `400` or higher is a failure.

However, I think we just want to confirm that the tenant's container is returning any HTTP response at all. If it's returning anything, error or not, we want the end user to access it.